### ISSUE TITLE
Improve/fix help

### ISSUE THREAD

**Issue by pawamoy** [timothycrosley/cruft#7]
There seems to be a problem with the help text :smile:   
Would be nice to get actual descriptions instead of variables types :stuck_out_tongue: 

```console
$ poetry run cruft link --help
usage: cruft [-h] [-p PROJECT_DIR] [-u] [-n] [-c CONFIG_FILE] [-d]
             [-e EXTRA_CONTEXT]
             template_git_url

Links an existing project created from a template, to the template it was
created from. Examples: link( 'https://github.com/timothycrosley/cookiecutter-
python/', no_input=True, use_latest=True ) -------

positional arguments:
  template_git_url      String

optional arguments:
  -h, --help            show this help message and exit
  -p PROJECT_DIR, --project_dir PROJECT_DIR
                        String
  -u, --use_latest      Boolean
  -n, --no_input        Boolean
  -c CONFIG_FILE, --config_file CONFIG_FILE
                        Union type; Union[X, Y] means either X or Y. To define
                        a union, use e.g. Union[int, str]. Details: - The
                        arguments must be types and there must be at least
                        one. - None as an argument is a special case and is
                        replaced by type(None). - Unions of unions are
                        flattened, e.g.:: Union[Union[int, str], float] ==
                        Union[int, str, float] - Unions of a single argument
                        vanish, e.g.:: Union[int] == int # The constructor
                        actually returns int - Redundant arguments are
                        skipped, e.g.:: Union[int, str, int] == Union[int,
                        str] - When comparing unions, the argument order is
                        ignored, e.g.:: Union[int, str] == Union[str, int] -
                        When two arguments have a subclass relationship, the
                        least derived argument is kept, e.g.:: class Employee:
                        pass class Manager(Employee): pass Union[int,
                        Employee, Manager] == Union[int, Employee]
                        Union[Manager, int, Employee] == Union[int, Employee]
                        Union[Employee, Manager] == Employee - Similar for
                        object:: Union[int, object] == object - You cannot
                        subclass or instantiate a union. - You can use
                        Optional[X] as a shorthand for Union[X, None].
  -d, --default_config  Boolean
  -e EXTRA_CONTEXT, --extra_context EXTRA_CONTEXT
                        Union type; Union[X, Y] means either X or Y. To define
                        a union, use e.g. Union[int, str]. Details: - The
                        arguments must be types and there must be at least
                        one. - None as an argument is a special case and is
                        replaced by type(None). - Unions of unions are
                        flattened, e.g.:: Union[Union[int, str], float] ==
                        Union[int, str, float] - Unions of a single argument
                        vanish, e.g.:: Union[int] == int # The constructor
                        actually returns int - Redundant arguments are
                        skipped, e.g.:: Union[int, str, int] == Union[int,
                        str] - When comparing unions, the argument order is
                        ignored, e.g.:: Union[int, str] == Union[str, int] -
                        When two arguments have a subclass relationship, the
                        least derived argument is kept, e.g.:: class Employee:
                        pass class Manager(Employee): pass Union[int,
                        Employee, Manager] == Union[int, Employee]
                        Union[Manager, int, Employee] == Union[int, Employee]
                        Union[Employee, Manager] == Employee - Similar for
                        object:: Union[int, object] == object - You cannot
                        subclass or instantiate a union. - You can use
                        Optional[X] as a shorthand for Union[X, None].
```

**Comment by pawamoy** [comment]
@timothycrosley any advice on how to fix this? I guess the help is auto-generated by hug?

**Comment by pawamoy** [comment]
I'm using copier now, closing :grimacing:

**Comment by samj1912** [comment]
Should be fixed by #31
